autofs-5.0.7 - depricate nosymlink pseudo option

From: Ian Kent <raven@themaw.net>

The undocumented "nosymlink" option was the only way to force local
NFS mounting until the more descriptive "nobind" option was added.

So depricate the "nosymlink" option in favour of the "nobind" option.
---
 CHANGELOG           |    1 +
 modules/mount_nfs.c |    4 ++++
 2 files changed, 5 insertions(+)

--- autofs-5.0.5.orig/CHANGELOG
+++ autofs-5.0.5/CHANGELOG
@@ -156,6 +156,7 @@
 - fix kernel verion check of version components.
 - fix submount tree not all expiring.
 - fix max() declaration.
+- depricate nosymlink pseudo option.
 
 03/09/2009 autofs-5.0.5
 -----------------------
--- autofs-5.0.5.orig/modules/mount_nfs.c
+++ autofs-5.0.5/modules/mount_nfs.c
@@ -131,6 +131,10 @@ int mount_mount(struct autofs_point *ap,
 				rdma = 1;
 
 			if (_strncmp("nosymlink", cp, o_len) == 0) {
+				warn(ap->logopt, MODPREFIX
+				     "the \"nosymlink\" option is depricated "
+				     "and will soon be removed, "
+				     "use the \"nobind\" option instead");
 				nosymlink = 1;
 			} else if (_strncmp("nobind", cp, o_len) == 0) {
 				nobind = 1;
